numba/ir_utils.py

```diff
@@ -1630,3 +1630,17 @@ def set_index_var_of_get_setitem(stmt, new_index):
     else:
         raise ValueError("getitem or setitem node expected but received {}".format(
                      stmt))
+
+def is_namedtuple_class(c):
+    """check if c is a namedtuple class"""
+    if not isinstance(c, type):
+        return False
+    # should have only tuple as superclass
+    bases = c.__bases__
+    if len(bases) != 1 or bases[0] != tuple:
+        return False
+    # should have _fields that is all string
```

Comment:
Also has a `_make`, is this worth checking for?

Reply:
done.